test/good-fixtures/RNASeqAnnDataZarrViewConfBuilder/fake-asct-http200-published-conf.json

```diff
@@ -1,122 +1,124 @@
-{
-  "coordinationSpace": {
-    "dataset": {
-      "A": "A"
+[
```

Comment:
We consistently return a list now. (Hide whitespace on the diff to make review easier.)